tests/admin_views/tests.py

```diff
@@ -41,21 +41,7 @@
 
 from . import customadmin
 from .admin import CityAdmin, site, site2
-from .models import (
-    Actor, AdminOrderedAdminMethod, AdminOrderedCallable, AdminOrderedField,
-    AdminOrderedModelMethod, Answer, Article, BarAccount, Book, Category,
-    Chapter, Character, Child, Choice, City, Collector, Color2,
-    ComplexSortedPerson, CoverLetter, CustomArticle, DooHickey, Employee,
-    EmptyModel, FancyDoodad, FieldOverridePost, FilteredManager, FooAccount,
-    FoodDelivery, FunkyTag, Gallery, Grommet, Inquisition, Language,
-    MainPrepopulated, ModelWithStringPrimaryKey, OtherStory, Paper, Parent,
-    ParentWithDependentChildren, Person, Persona, Picture, Pizza, Plot,
-    PluggableSearchPerson, Podcast, Post, Promo, Question, RelatedPrepopulated,
-    Report, Restaurant, RowLevelChangePermissionModel, Section, ShortMessage,
-    Simple, Story, Subscriber, Telegram, Topping, UnchangeableObject,
-    UndeletableObject, UnorderedObject, Vodcast, Whatsit, Widget, Worker,
-    WorkHour,
-)
+from .models import *  # NOQA
```

Comment:
I'd like not to use `import *` as that makes flake8 unable to detect undefined names. I think a better long term solution is to break this file up into multiple `test_*.py` files so it's not so monolithic.